app/Http/Controllers/Api/PagesController.php

```diff
@@ -84,19 +85,24 @@ public function postNewAppDetails(Request $request)
             'description'  => 'required',
         ]);
 
-        $returnData = AppDetail::create([
-            'name'         => $request->name,
-            'user_id'      => auth()->user()->id,
-            'homepage_url' => $request->homepage_url,
-            'description'  => $request->description,
-            'api_token'    => $this->generateToken(),
-        ]);
+        try {
+            $returnData = AppDetail::create([
```

Comment:
**Code Quality-NitPick**
@andela-araimi The try and catch here is not needed, `QueryException` could be any kind of Query error. It will be better and easier if you do a check in the DB if the homepage_url is already there, then you send the error message that `App already created`